### DISCUSSION TITLE
This app has become too bloated

### DISCUSSION THREAD

**Discussion by pelaird** [turkaysoft/glow#49] [Ideas]
This was a great app, but it has become too bloated for my use. It's trying to do too much and is trying to be everything to everyone. I'm back to using Speccy.

**Comment by turkaysoft** [comment]
> This was a great app, but it has become too bloated for my use. It's trying to do too much and is trying to be everything to everyone. I'm back to using Speccy.

Glow continues to evolve with each release. Even as the number of features grows, the program is more optimized and faster. What exactly is the problem?
